pkg/apis/core/validation/validation.go

```diff
@@ -5939,7 +5939,7 @@ func ValidateSecurityContext(sc *core.SecurityContext, fldPath *field.Path) fiel
 
 		if sc.Capabilities != nil {
 			for _, cap := range sc.Capabilities.Add {
-				if string(cap) == "CAP_SYS_ADMIN" {
+				if string(cap) == "SYS_ADMIN" {
 					allErrs = append(allErrs, field.Invalid(fldPath, sc, "cannot set `allowPrivilegeEscalation` to false and `capabilities.Add` CAP_SYS_ADMIN"))
```

Comment:
Maybe the error string needs to be 

"cannot set `allowPrivilegeEscalation` to false and `capabilities.Add` SYS_ADMIN"